app/models/schedule.rb

```diff
@@ -1,7 +1,6 @@
 class Schedule < ActiveRecord::Base
   include RankedModel
 
-  has_many :volunteers, through: :schedule_volunteers
```

Comment:
schedule.volunteers is in many places in the app. Are those ALL of class ScheduleChain?